swift/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# Use the latest Swift from Docker Hub
+FROM ibmcom/swift-ubuntu
+
+# Add current application
+ADD . .
```

Comment:
we typically add things to an /app folder in the container, may be nice to be consistent :)